onnxmltools/convert/coreml/convert.py

```diff
@@ -18,7 +18,7 @@
 from .shape_calculators import neural_network as nn_shape_calculators
 
 
-def convert(model, name=None, initial_types=None, doc_string='', targeted_onnx=onnx.__version__):
+def convert(model, name=None, initial_types=None, doc_string='', targeted_onnx=onnx.__version__, custom_conversion={}):
```

Comment:
Need to extend the subsequent doc string to cover the new argument, `custom_conversion`. Also, `custom_conversion"s"` is a more accurate name, I guess.
In addition, why `costom_conversion` is not used inside this function?